src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -101,6 +101,9 @@ void Terminal::UpdateSettings(ICoreSettings settings)
         GetRenderSettings().SetColorTableEntry(TextColor::FRAME_BACKGROUND, til::color{ settings.TabColor().Value() });
     }
 
+    // Save the changes made above and in UpdateAppearance as the new default render settings.
+    GetRenderSettings().SaveDefaultSettings();

```

Comment:
If the code above calls `SetColorTableEntry`, won't this overwrite the modified colors set by the terminal application (as if a RIS was received)?

Reply:
Yeah. That's typically what you want if you're actually using the settings to change the color scheme, but not so much when you're editing an unrelated setting. In fact the colors can even end up being reset when when you haven't edited the settings at all! This is a known problem that is tracked in issue #11522.

Reply:
> Yeah. That's typically what you want if you're actually using the settings to change the color scheme

I'm somewhat surprised about that - I expected the opposite. But that answered my question. It was the only concern I had, and otherwise the PR looks great. :)

Reply:
Just to be clear, I don't feel strongly about it either way, but that's the way it's always worked, and it seemed to me a reasonable behavior. If you open up a shell with a particular color scheme, decide you don't like it, so you change the scheme in the settings, wouldn't you expect that shell to update straight away?

Of course if you were busy using an application that set its own palette, having it overwritten would be annoying, but it's much less likely you'd be trying to switch your color scheme in that situation (this assumes that #11522 is fixed, so changing other settings wouldn't trigger a color scheme refresh).

Either way, if we want to reconsider that behavior, I think it's best raised as a separate issue.